addons/io_scene_gltf2_msfs/com/material/msfs_material_panel.py

```diff
@@ -247,5 +198,58 @@ def draw(self, context):
                     row.prop(mat,'msfs_uv_clamp_y')
                     #row.prop(mat,'msfs_uv_clamp_z')    - removed by Asobo, probably because it never made sense in the first place.
 
+            if (mat.msfs_show_albedo == True or mat.msfs_show_metallic == True or mat.msfs_show_normal == True or mat.msfs_show_emissive == True or mat.msfs_show_detail_albedo == True or 
+                mat.msfs_show_detail_metallic == True or mat.msfs_show_detail_normal == True or mat.msfs_show_blend_mask == True or mat.msfs_show_anisotropic_direction == True or
+                mat.msfs_show_clearcoat == True or mat.msfs_show_behind_glass == True or mat.msfs_show_wiper_mask == True):
+
+                box = layout.box()
+                box.label(text="Texture maps",icon='TEXTURE')
+                if mat.msfs_show_albedo == True:
+                    box.label(text = "Albedo:")
```

Comment:
For all these labels, you can remove the `:` as Blender adds that when drawing the UI, so currently there are two colons